astrokat/observe_main.py

```diff
@@ -642,7 +642,7 @@ def main(args):
     # astrokat does not support observational command line parameters
     # all observational parameters must be in the YAML file
     # command line arguments will be dropped to unknown arguments
-    unknown_args = [arg for arg in args if arg.startswith('--')]
+    unknown_args = [arg for arg in args if arg and arg.startswith('--')]
```

Comment:
Are some args `None` or some other value that evaluates to false?  When does that occur?

Reply:
When `os.getenv("sb_code_id")` returns `None`

https://github.com/ska-sa/astrokat/blob/114c75da636d6b09be71b6a6c1c9f8e1cf03d0f5/astrokat/test/test_offline_observe.py#L55

Reply:
`None` isn't something you could get as a command line argument.  Rather fix the tests so they don't pass invalid arguments.

Reply:
Come to think of it, make sense that they should fail. Will fix.